### ISSUE TITLE
add verbose output option for debugging purposes

### ISSUE THREAD

**Issue by longjoel** [nicoespeon/sass-graph-viz#7]
When I try to run it on the master scss file from my SASS folder, it opens a new browser window to start displaying output, then the command line tool closes with no output to help me figure out what went wrong. I would like to see output.

**Comment by nicoespeon** [comment]
Hi @longjoel 👋 

That's a good suggestion indeed. I will do something so we can debug your use case.

Thanks for the feedback.

**Comment by nicoespeon** [comment]
Hi @longjoel,

I just released [v2.2.0](https://github.com/nicoespeon/sass-graph-viz/releases/tag/v2.2.0) that includes a `-d, --debug` option to the CLI. Using it, it should log information about the process.

Example of output running `sgv -d examples/basic-tree/fixtures` in the project:

```
Generate visual graph for:
/Users/espeon/Development/github/nicoespeon/sass-graph-viz/examples/basic-tree/fixtures

Resolved target folder is:
/Users/espeon/Development/github/nicoespeon/sass-graph-viz/examples/basic-tree/fixtures

Generated graph from target folder is:
{
  "main": [
    "_base",
    "_footer",
    "_header"
  ],
  "_base": [],
  "_footer": [
    "_colors"
  ],
  "_colors": [],
  "_header": [
    "_colors"
  ]
}

Render graph through port: 3000
```

I hope that would help understand better what is going wrong. Let me know about that 😉

**Comment by longjoel** [comment]
Thanks!
